context/AuthContextProvider.tsx

```diff
@@ -97,6 +107,9 @@ export const AuthContextProvider = ({
 
     try {
       const user = await account.get();
+      if (user.labels.includes('admin')) {
+        setIsSuperAdmin(true);
+      }
```

Comment:
I'm not sure I understand the difference between super admin and admin. From what it seems, super admin has all the access that admin does, correct? If so, what differentiates these two roles?

Reply:
Great question! 

## SUPER ADMIN
Super Admin is US the Developers.  This is controlled by adding the `admin` field in the labels column for the user:

![CleanShot 2024-08-08 at 22 20 22](https://github.com/user-attachments/assets/b022fa39-6dbd-4bc6-a9b3-11691572013e)

## ADMIN
Admin is for a specific league (which we have not implemented yet).  The creator of the league is the admin for that league only and should be able to access `/admin` routes.  That admin can also assign others in the league to the `admin` role.